Lib/test/test_tools/test_sundry.py

```diff
@@ -17,8 +17,6 @@ class TestSundryScripts(unittest.TestCase):
     # At least make sure the rest don't have syntax errors.  When tests are
     # added for a script it should be added to the allowlist below.
 
```

Comment:
This comment is confusing without declaration of `skiplist`. We need to either remove it too or return `skiplist = []` with `if name in self.skiplist:...`.

I prefer the former because the only additions into the list were in 2018 (`2to3`) and 2014 (test_sundry itself). See https://github.com/python/cpython/commits/main/Lib/test/test_tools/test_sundry.py.

Reply:
Comment removed